.palantir/revapi.yml

```diff
@@ -873,6 +873,10 @@ acceptedBreaks:
       new: "method void org.apache.iceberg.encryption.Ciphers::<init>()"
       justification: "Static utility class - should not have public constructor"
   "1.4.0":
+    org.apache.iceberg:iceberg-api:
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.OverwriteFiles org.apache.iceberg.OverwriteFiles::appendManifest(org.apache.iceberg.ManifestFile)"
+      justification: "Provision to use manifestFile with thousand of data files together"
```

Comment:
Adding new things should not be a breaking change if you provide it as a default interface. 

Can you try making it as default interface which throws UnsupportedOperationException?